lbz/authz.py

```diff
@@ -87,44 +48,46 @@ def validate(self, function_name):
         if self.outcome == DENY:
             raise PermissionDenied
 
-    def set_initial_state(self, function_name: str) -> None:
-        self.outcome = DENY
-        self.action = self._permissions[function_name]
-
-    def set_policy(self, token: str):
-        policy = decode_jwt(token)
-        self.allow = policy["allow"]
-        self.deny = policy["deny"]
-        self.expiration = policy.get(EXPIRATION_KEY)
-        self.iss = policy.get("iss")
-        self.outcome = DENY
-        self.allowed_resource = None
-        self.denied_resource = None
+    def _set_policy(self, auth_jwt: str):
```

Comment:
Could you please move it right after the `__init__` ? I think that will make readeing the code faster.

Reply:
I'll move it after `__repr__`, I think it's good to keep the `__x__` methods in the beginning of a class